### DISCUSSION TITLE
Cancelling billing plan

### DISCUSSION THREAD

**Discussion by KPham4588** [vercel/community#5235] [Account Issue]
### Question

Hello, I tried to cancel my pro plan and Vercel required me to delete my team. So I deleted the project, then deleted my team. Then now when I go to https://vercel.com/dashboard -> https://vercel.com/account I get a 404 error with no option to see my billing. How can I make sure I am no longer getting billed?

**Comment by amyegan** [comment]
Hi @KPham4588. I can't discuss your account details here in the interest of protecting your private information. If you haven't already, please [submit a case](https://vercel.com/help#issues) to the Customer Success Team with the `Invoice Enquiry` selected as the problem area. That will connect you with the specialists who can check your account's billing and invoice info.
